zellij-server/src/panes/terminal_pane.rs

```diff
@@ -398,12 +398,13 @@ impl Pane for TerminalPane {
             frame.add_exit_status(exit_status.as_ref().copied());
         }
 
-        match self.frame.get(&client_id) {
+        let err_context = || format!("failed to render frame for client {client_id}");
```

Comment:
Could you please move this to the very top of the function? It makes spotting whether someone already added an `err_context` much easier.